### PR TITLE
Update Analytics for Microgateway documentation

### PR DESCRIPTION
**Pull request by HeshanSudarshana**
### Purpose
This PR updates the Analytics for Microgateway documentation with the new updates and the formatting issues.

Fixes https://github.com/wso2/docs-mg/issues/16

### CODE REVIEW

File: en/docs/how-tos/analytics-for-microgateway.md [hunk 1]
```diff
@@ -60,7 +60,13 @@ Real-Time data publishing has the following advantages:
...
-The following sections describe how to configure the WSO2 API-M Analytics Server for Microgateway.
+The following sections describe how to configure the WSO2 API-M Analytics Server for Microgateway. When adding the relevant analytics configurations to `<MICROGW_HOME>/conf/micro-gw.conf` file, make sure to add the version of the API-M Analytics you are using from the following config.
+
+```toml
+[analytics]
+  # The configured API Manager analytics stream version
+  streamVersion = "3.2.0"
+```
 
 #### Using periodical data publishing (file-based)
```
Comment (Rajith90) [inline en/docs/how-tos/analytics-for-microgateway.md:71]: Lets metion here this config is only required if you are choosing the file based analytics publishing

File: en/docs/how-tos/analytics-for-microgateway.md [hunk 2]
```diff
@@ -203,10 +215,10 @@ The configurations are described in the table below.
```
Comment (Rajith90) [inline en/docs/how-tos/analytics-for-microgateway.md:215]: Lets also mention here these steps are required only if real time publishing is used. And methiion we have to configure only one of them not the both

**Comment by Mariangela** [comment]
@HeshanSudarshana, have you addressed the review comments in this PR? Also pls sort out the merge conflict.

File: en/docs/how-tos/analytics-for-microgateway.md (hunk 2 above)
Comment (HeshanSudarshana) [reply en/docs/how-tos/analytics-for-microgateway.md:227]: Done with https://github.com/wso2/docs-mg/pull/124/commits/d6479400b0cf6238740347e88d41211234f72980

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@HeshanSudarshana , please send these changes to the APIM 4.0.0 MGW related docs if applicable.

**Pull request merged**